panels/automated_emails.py

```diff
@@ -1,7 +1,12 @@
 from panels import *
 
-AutomatedEmail.queries[PanelApplication] = lambda session: session.query(PanelApplication) \
-    .options(subqueryload(PanelApplication.applicants).subqueryload(PanelApplicant.attendee))
+
+AutomatedEmail.queries[PanelApplication] = lambda session: session \
+    .query(PanelApplication) \
+    .options(
+        subqueryload(PanelApplication.applicants)
+            .subqueryload(PanelApplicant.attendee)) \
+    .order_by(PanelApplication.id)
```

Comment:
In short, `order_by` is required whenever you are using `subqueryload()`. Without deterministic ordering, you can end up with the wrong objects in your relationship attributes. From a performance perspective, it's negligible because `id` is an indexed column. I refer you the following docs:
* [The Importance of Ordering](http://docs.sqlalchemy.org/en/latest/orm/loading_relationships.html#the-importance-of-ordering)
* [Why is ORDER BY required with LIMIT (especially with subqueryload())?](http://docs.sqlalchemy.org/en/latest/faq/ormconfiguration.html#why-is-order-by-required-with-limit-especially-with-subqueryload)

Reply:
Ahh, good to know.  I usually end up using ``joinedload``, but I'll definitely have to keep this in mind for those occasions where I have enough levels of nested relationships that I need subqueryload.

Reply:
I know each `subqueryload` adds an additional `select` statement, but the queries generated by `subqueryload` are almost always more efficient than the one's generated by `joinedload`. I know there's a way to speed up `joinedloads` by using `innerjoin=True`, if you're loading a non-nullable relationship (as [described here](http://docs.sqlalchemy.org/en/latest/orm/loading_relationships.html#joined-eager-loading)). I've never experimented with that, but maybe I should give it a try!

Reply:
Hmm, interesting.  I've recently implemented some automated performance measurements on a recent microservice we implemented at work, so that would be a good opportunity to try switching from ``joinedload`` to ``subqueryload`` and see what effect that has on the overall performance of the API, since the most performance-critical pieces all have some kind of ``joinedload``.

Thanks for the tip!

Reply:
Definitely let me know the results (and if `innerjoin=True` makes a difference)!